docs/source/classroom/class_guide.md

```diff
@@ -54,7 +54,7 @@ header-rows: 1
   - 256 GB
 * - courses-gpu
   - 4 hr / 8 hrs
-  - 8
+  - 4
```

Comment:
I thought the courses-gpu was changed to 2 but I might be misremembering this.

Reply:
you're right. it says 2 in my notes. I just fixed it here as well.